ext/js/data/options-util.js

```diff
@@ -495,6 +496,7 @@ export class OptionsUtil {
      * @returns {import('options-util').UpdateFunction[]}
      */
     _getVersionUpdates(targetVersion) {
+        /* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
Huh, why is this needed? Are we intentionally using an unbound `this` here? If so it'd probably be better to take in an arg or something

Reply:
None of these are bound because they are bound later on, as it is used purely internally.

https://github.com/themoeway/yomitan/blob/805cf9cb3ea744a6d7c0a5da27fc9ef4e6f08626/ext/js/data/options-util.js#L485